WooCommerce/Classes/POS/Presentation/PaymentButtons.swift

```diff
@@ -0,0 +1,96 @@
+import SwiftUI
+
+struct PaymentsActionButtons: View {
+    @EnvironmentObject private var posModel: PointOfSaleAggregateModel
+    @State private var isShowingSendReceiptModal: Bool = false
+
+    private var shouldShowSendReceiptButton: Bool {
+        ServiceLocator.featureFlagService.isFeatureFlagEnabled(.sendReceiptsForPointOfSale)
+    }
+
+    var body: some View {
+        VStack {
+            sendReceiptButton
+                .renderedIf(shouldShowSendReceiptButton)
+            newOrderButton
+        }
+        .posModal(isPresented: $isShowingSendReceiptModal) {
+            POSSendReceiptModalView(sendReceipt: { email in
+                Task { @MainActor in
+                    await posModel.sendReceipt(to: email)
+                }
+            }, isPresented: $isShowingSendReceiptModal)
+            .posModalSizing()
+        }
+    }
+}
+
+#if DEBUG
+#Preview {
+    let posModel = PointOfSaleAggregateModel(
+        itemsController: PointOfSalePreviewItemsController(),
+        cardPresentPaymentService: CardPresentPaymentPreviewService(),
+        orderController: PointOfSalePreviewOrderController())
+    PaymentsActionButtons()
+        .environmentObject(posModel)
+}
+#endif
+
+private extension PaymentsActionButtons {
+    var sendReceiptButton: some View {
+        Button(action: {
+            isShowingSendReceiptModal = true
+        }, label: {
+            HStack(spacing: Constants.buttonSpacing) {
+                Text(Localization.sendReceipt)
+                    .font(Constants.buttonFont)
+            }
+            .frame(minWidth: UIScreen.main.bounds.width / 2)
+        })
+        .padding(Constants.buttonPadding)
+        .foregroundColor(Color.posPrimaryText)
+        .background(Color.clear)
+        .overlay {
+            RoundedRectangle(cornerRadius: Constants.buttonCornerRadius)
+                        .stroke(Color.posPrimaryText, lineWidth: 1.0)
+        }
+    }
+
+    var newOrderButton: some View {
+        Button(action: {
+            posModel.startNewCart()
+        }, label: {
+            HStack(spacing: Constants.buttonSpacing) {
+                Text(Localization.newOrder)
+                    .font(Constants.buttonFont)
+            }
+            .frame(minWidth: UIScreen.main.bounds.width / 2)
+        })
+        .padding(Constants.buttonPadding)
+        .foregroundColor(Color.posPrimaryTextInverted)
+        .background(Color.posOverlayFillInverted)
+        .cornerRadius(Constants.buttonCornerRadius)
+    }
+}
+
+extension PaymentsActionButtons {
+    enum Constants {
+        static let paymentsButtonSpacing: CGFloat = 80
+        static let paymentsButtonButtonSpacingAnimationDelay: CGFloat = 0.3
```

Comment:
They don't, good catch. We don't need them anymore since we use the matched geometry to animate them. Removed on https://github.com/woocommerce/woocommerce-ios/pull/14622/commits/a2c09f585cb4291a15d495a18dc50a6ec7fcb766